src/otg/preprocess/finngen/summary_stats.py

```diff
@@ -0,0 +1,121 @@
+"""Datasource ingestion: FinnGen."""
+
+from __future__ import annotations
+
+import logging
+import sys
+from dataclasses import dataclass
+from typing import TYPE_CHECKING
+
+import pyspark.sql.functions as f
+import pyspark.sql.types as t
+from pyspark.sql import SparkSession
+
+from otg.common.schemas import parse_spark_schema
+from otg.common.utils import calculate_confidence_interval, parse_pvalue
+from otg.dataset.summary_statistics import SummaryStatistics
+
+if TYPE_CHECKING:
+    from pyspark.sql import DataFrame
+    from pyspark.sql.types import StructType
+
+logging.basicConfig(level=logging.INFO, format="%(asctime)s %(message)s")
+
+
+@dataclass
+class FinnGenSummaryStats(SummaryStatistics):
+    """Summary statistics dataset for FinnGen."""
+
+    @classmethod
+    def get_schema(cls: type[SummaryStatistics]) -> StructType:
```

Comment:
not needed. you get it by inheritance from SummaryStatistics

Reply:
You're right, fixed.